.travis.yml

```diff
@@ -1,12 +1,22 @@
 --- 
 services: 
   - docker
-before_install: 
+before_script: 
   - "export PYTHONPATH=$PYTHONPATH:$(pwd)"
   - "docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD"
-  - "docker --version"
-  - "docker build --target test --tag todo-app:test ."
-
-script: 
-  - "echo \"yes\""
-  - "docker run -it --rm --env TRELLO_TOKEN=$TRELLO_TOKEN --env TRELLO_KEY=$TRELLO_KEY --env FLASK_ENV=test todo-app:test tests"
\ No newline at end of file
+  - docker login -u=_ -p=$HEROKU_API_KEY registry.heroku.com
```

Comment:
For both of these login steps, we should ideally action them using the `--password-stdin` flag rather than passing the password explicitly, to avoid leaking those credentials to shell logs. [See the docs](https://docs.docker.com/engine/reference/commandline/login/) for more info, example of how that might look below:
```
echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME--password-stdin
```

Reply:
Separately, we could also move the heroku login to a `before_deploy` lifecycle stage (or as part of your `deploy.sh`) since it's only relevant in the case that you're deploying

([That stage is mentioned in the docs here](https://docs.travis-ci.com/user/job-lifecycle/#deploying-your-code))